lib/bns/use_cases/use_cases.rb

```diff
@@ -148,4 +152,58 @@ def self.notify_pto_from_notion_to_discord(options)
 
     UseCases::UseCase.new(use_case_cofig)
   end
+
+  # Provides an instance of the PTO notifications from Postgres to Slack use case implementation.
+  #
+  # <br>
+  # <b>Example</b>
+  #
+  # options = {
+  #   fetch_options: {
+  #     connection: {
+  #       host: "localhost",
+  #       port: 5432,
+  #       dbname: "db_pto",
+  #       user: "postgres",
+  #       password: "postgres"
+  #     },
+  #     query: "SELECT * FROM db_pto"
+  #   },
+  #   dispatch_options:{
+  #     webhook: "https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX",
+  #     name: "Pto Bot"
+  #   },
+  #   format_options: {
+  #     template: "Custom template",
+  #     timezone: "-05:00"
+  #   }
+  # }
+  #
+  #   use_case = UseCases.notify_pto_from_postgres_to_slack(options)
+  #   use_case.perform
+  #
+  #   #################################################################################
+  #
+  #   Requirements:
+  #   * A connection to a Postgres database and a table with the following structure:
+  #
+  #         ________________________________________________________________________________________________________
+  #         |      Name (person)   |        start_date                       |       end_date                       |
```

Comment:
What do you think if you rather include here the description of the table using the `DESCRIBE` statement?